src/shared/browser/__mocks__/chrome.ts

```diff
@@ -0,0 +1,51 @@
+import { MockedObject, vi } from 'vitest'
+
+type IChrome = typeof chrome
+type StoreValue = Record<string, unknown>
+
+let store: StoreValue = {}
+
+function resolveKey(key: unknown) {
+  if (typeof key === 'string') {
+    const result: StoreValue = {}
+    result[key] = store[key]
+    return result
+  } else if (Array.isArray(key)) {
+    return key.reduce((acc, curr) => {
+      acc[curr] = store[curr]
+      return acc
+    }, {})
+  }
+  throw new Error('Wrong key given')
```

Comment:
Let's make more meaningful message. Something like `Key expected to be string or string[], but {typeof key} was provided: {key}`